File: src/main/java/ru/ioleksiv/telegram/bot/api/model/annotation/validator/impl/MessageFilterValidator.java
package ru.ioleksiv.telegram.bot.api.model.annotation.validator.impl;

import ru.ioleksiv.telegram.bot.api.model.annotation.validator.CustomValidator;
import ru.ioleksiv.telegram.bot.api.model.annotation.validator.FilterValidator;
import ru.ioleksiv.telegram.bot.api.model.objects.std.Message;

public class MessageFilterValidator extends FilterValidator<Message> {

    public MessageFilterValidator(CustomValidator<Message> validator) {
        super(Message.class, validator);
    }
}

File: core/src/main/kotlin/io/github/oleksivio/tl/kbot/core/model/annotation/validator/impl/MessageFilterValidator.kt
package io.github.oleksivio.tl.kbot.core.model.annotation.validator.impl

import io.github.oleksivio.tl.kbot.core.model.annotation.validator.FilterValidator
import io.github.oleksivio.tl.kbot.server.api.objects.std.Message
import io.github.oleksivio.tl.kbot.core.core.controller.handler.check.Validator

class MessageFilterValidator(validator: Validator<Message>) : FilterValidator<Message>(
    Message::class, validator)
